### PR TITLE
#168: replaced all remaining mockito instances of Buckets with MkBuckets

### PR DESCRIPTION
**Pull request by ggajos**
for #168
replaced all remaining mockito instances of Buckets with MkBuckets

### CODE REVIEW

**Comment by karato** [comment]
Thanks for your pull request, let me find someone who can review it

**Comment by karato** [comment]
@alevohin review it

**Comment by alevohin** [comment]
@ggajos class `AwsDocsTest` contains `final Bucket bucket = Mockito.mock(Bucket.class);` but puzzle ask to

> Replace all mocks

File: src/test/java/com/nerodesk/om/aws/AwsUserTest.java [hunk 1]
```diff
@@ -53,7 +62,10 @@
     @Test
     public void obtainsDocs() throws Exception {
         MatcherAssert.assertThat(
-            new AwsUser(Mockito.mock(Bucket.class), "user1").docs(),
+            new AwsUser(
+                new MkBucket(this.folder.newFile(), "1"),
```
Comment (alevohin) [inline src/test/java/com/nerodesk/om/aws/AwsUserTest.java]: "user1" could be placed at this line according to http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#indentation (last paragraph)

**Comment by alevohin** [comment]
@ggajos thank you! Please see comments above

**Comment by ggajos** [comment]
@alevohin there is big TODO above this mock you mention (AwsDocsTest), that is why I didn't touch this test

**Comment by ggajos** [comment]
@alevohin thanks for feedback, please have a look if everything is fine now

**Comment by alevohin** [comment]
> @alevohin there is big TODO above this mock you mention (AwsDocsTest), that is why I didn't touch this test

@ggajos you are totally right! There is puzzle `@todo #131:30min` with explanation why we should use mock there.

**Comment by alevohin** [comment]
@rultor merge

**Comment by yegor256** [comment]
@rultor merge

**Comment by karato** [comment]
@alevohin thank you, added **19 mins** to your acc, payment num is `55065359`, 98 hours and 22 mins spent in total to complete

review comments (c=4) added as a bonus

+19 added to your rating, at the moment it is: [+2157](http://www.netbout.com/b/35105?open=rating)

**Comment by karato** [comment]
@rultor deploy now pls

**Pull request merged**